# Existing set of patches from the desktop gate for manpages
# Not suitable for upstream

/jds/bin/diff -uprN OpenSP-1.5.1.old/nsgmls.1 OpenSP-1.5.1/nsgmls.1
--- OpenSP-1.5.1.old/nsgmls.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/nsgmls.1	2009-02-20 17:08:34.123395000 +0000
@@ -0,0 +1,1 @@
+.so man1/onsgmls.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/onsgmls.1 OpenSP-1.5.1/onsgmls.1
--- OpenSP-1.5.1.old/osgml2xml.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/osgml2xml.1	2009-02-20 17:09:47.763849000 +0000
@@ -0,0 +1,1 @@
+.so man1/osx.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/osgmlnorm.1 OpenSP-1.5.1/osgmlnorm.1
--- OpenSP-1.5.1.old/s2x.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/s2x.1	2009-02-20 17:09:22.032465000 +0000
@@ -0,0 +1,1 @@
+.so man1/osx.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/sgml2xml.1 OpenSP-1.5.1/sgml2xml.1
--- OpenSP-1.5.1.old/sgml2xml.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/sgml2xml.1	2009-02-20 17:06:32.438505000 +0000
@@ -0,0 +1,1 @@
+.so man1/osx.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/sgmlnorm.1 OpenSP-1.5.1/sgmlnorm.1
--- OpenSP-1.5.1.old/sgmlnorm.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/sgmlnorm.1	2009-02-20 17:07:24.832231000 +0000
@@ -0,0 +1,1 @@
+.so man1/osgmlnorm.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/spam.1 OpenSP-1.5.1/spam.1
--- OpenSP-1.5.1.old/spam.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/spam.1	2009-02-20 17:07:40.493548000 +0000
@@ -0,0 +1,1 @@
+.so man1/ospam.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/spcat.1 OpenSP-1.5.1/spcat.1
--- OpenSP-1.5.1.old/spcat.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/spcat.1	2009-02-20 17:08:04.384468000 +0000
@@ -0,0 +1,1 @@
+.so man1/ospcat.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/spent.1 OpenSP-1.5.1/spent.1
--- OpenSP-1.5.1.old/spent.1	1970-01-01 01:00:00.000000000 +0100
+++ OpenSP-1.5.1/spent.1	2009-02-20 17:08:10.105986000 +0000
@@ -0,0 +1,1 @@
+.so man1/ospent.1
/jds/bin/diff -uprN OpenSP-1.5.1.old/Makefile.am OpenSP-1.5.1/Makefile.am
--- OpenSP-1.5.1.old/Makefile.am	2009-02-20 16:56:54.307688000 +0000
+++ OpenSP-1.5.1/Makefile.am	2009-02-20 17:53:27.579799000 +0000
@@ -13,7 +13,7 @@ docdir = $(datadir)/doc
 pkgdocdir = $(docdir)/@PACKAGE@
 man1dir=$(mandir)/man1
 
-man_MANS = onsgmls.1 osgmlnorm.1 ospent.1 ospam.1 osx.1 ospcat.1
+man_MANS = nsgmls.1 onsgmls.1 osgml2xml.1 osgmlnorm.1 ospent.1 ospam.1 osx.1 ospcat.1 s2x.1 sgml2xml.1 sgmlnorm.1 spam.1 spcat.1 spent.1
 
 pkgdoc_DATA = README ABOUT-NLS NEWS COPYING AUTHORS BUGS ChangeLog \
 	releasenotes.html releasenotes.pdf releasenotes.ps
@@ -29,8 +29,7 @@ MAINTAINERCLEANFILES = install-sh mkinst
 ## We include msggen.pl, since it won't be generated on Windows.
 EXTRA_DIST = instmac.pl msggen.pl sunfix.sh SP.dsw SP.mak \
 	build-win32.bat sp-generate.mak .cvsignore all \
-	@PACKAGE@-@VERSION@.spec config.rpath onsgmls.1 \
-	osgmlnorm.1 ospam.1 ospent.1 osx.1 ospcat.1 \
+	@PACKAGE@-@VERSION@.spec config.rpath $(man_MANS) \
 	releasenotes.html releasenotes.pdf releasenotes.ps \
 	BUGS
 
